short_answer/static/js/src/short_answer.js

```diff
@@ -1,6 +1,13 @@
 /* Javascript for ShortAnswerXBlock. */
 function ShortAnswerXBlock(runtime, element, options) {
+  const $answerSubmitButton = $('.answer-submission-button', element);
+  const $shortAnswer = $('.short-answer', element);
+  const gradesPublished = options.gradesPublished;
   const maxScore = options.maximumScore;
+  const passedDue = options.passedDue;
+  const score = options.score;
+  // Variables that depend on other variables.
```

Comment:
All I can see are constants below :poop:

Reply:
There is only one variable for now but there could be more.

Reply:
https://en.wikipedia.org/wiki/Constant_(computer_programming)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const

Reply:
variables/constants 👍

Reply:
For the record:
https://docs.microsoft.com/en-us/scripting/javascript/reference/const-statement-javascript
https://blog.mariusschulz.com/2015/12/31/constant-variables-in-javascript-or-when-const-isnt-constant

Reply:
Goes to show how great this language is :smile:

Reply:
They are going places.